### ISSUE TITLE
AVT 3 - React FileUploader, Add Files button does not get proper focus and fails to open dialog

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3439]
## Environment
iOS 12.3.1 with VoiceOver and Safari

## Description
1. Swipe Right to Navigate to Add Files Button, VO announces, "Add Files button" but the button does not get focus. See screenshot with focus indicator as a small circle above the word Upload.
![IMG_0537](https://user-images.githubusercontent.com/21676914/61411679-7eff1b80-a8ac-11e9-9ec3-8025a5234df6.PNG)
2. Attempting to press Enter or Space when VO announces the Add file button does not open the dialog to select a file. 
**Expected Result:** The dialog should open, the user should be able to select the jpg image and the selected image should display.

**Comment by emyarod** [comment]
I believe this is an iOS bug, but just to confirm: could you try out this reduced test case?

https://2oub9.csb.app/

source https://codesandbox.io/s/trusting-wright-2oub9

**Comment by joshblack** [comment]
Hi @snidersd! 👋 Trying to reproduce now, it seems like the focus of the screen reader moves to the button in VO on iOS and Safari and correctly opens the dialog. This might have been addressed in a recent release! Let us know if this is still a problem and we'd be happy to re-open 😄

**Comment by snidersd** [comment]
@joshblack Retested with iOS 13.2.3 and its working as expected.
